src/slack.coffee

```diff
@@ -242,4 +245,29 @@ class SlackBot extends Adapter
     channel = @client.getChannelGroupOrDMByName envelope.room
     channel.setTopic strings.join "\n"
 
+  customMessage: (data) =>
+    channel = @client.getChannelGroupOrDMByName data.message.envelope.room
+
+    msg = []
+    data.attachments = data.content if data.content and not data.attachments
+    data.attachments = [data.attachments] unless Array.isArray data.attachments
```

Comment:
I've just updated the pull request in 471e71dea0acdf000f1fb0dfa621156d61148e8e to fix all of these issues. Thanks for pointing them out!